src/services/tiers.service.ts

```diff
@@ -154,10 +154,10 @@ export class TiersService {
   async applyTier(
     userWithEmail: { email: string; uuid: User['uuid'] },
     customer: Stripe.Customer,
-    lineItem: Stripe.InvoiceLineItem,
+    amountOfSeats: Stripe.InvoiceLineItem['quantity'],
     productId: string,
+    excludedServices?: Service[],
```

Comment:
I understand why you need this one, but it would better, at least, to rename this variable to `alreadyEnabledServices`